<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Widget -----------------------------------------*/
QWidget {
	background-color: #2b2b2b;
    outline: 0px;
}

/* Tree -----------------------------------------*/

QTreeView {
	background-color: #333333;
    outline: 0px;
    border: 1px solid rgb(117, 118, 118);
}

QTreeView::item {
    color:#d4d4d4;
	/*background-color: #323232;*/
	height:30px;

}

QTreeView::item:selected {
    background-color: #5285a6;
}

QHeaderView::section {
    font: 10pt &quot;Consolas&quot;;
    height:25px;
    color:#FFFFFF;
    background:#5285a6;
    border-left:0px solid gray;
    border-right:1px solid gray;
    border-top:0px solid gray;
    border-bottom:0px solid gray;
}
/* LsitView -----------------------------------------*/
QListView {
	background-color: #333333;
    outline: 0px;
    border: 1px solid rgb(117, 118, 118);
}

/* Label -----------------------------------------*/
QLabel {
	color: #c8c8c8;
	font: 11pt &quot;Consolas&quot;;
}

/* LineEdit -----------------------------------------*/
QLineEdit {
    color: #c8c8c8;
    font: 11pt &quot;Consolas&quot;;
    border-radius: 5px;
    border: 1px solid rgb(117, 118, 118);
    background-color: #36393f;

}


/* QPushButton -----------------------------------------*/
QPushButton{
	color: #c8c8c8;
    font: 11pt &quot;Consolas&quot;;
    border-radius: 5px;
    border: 1px solid rgb(117, 118, 118);
    background-color: #2d2d2d;
    background-repeat: no-repeat;
    background-position: left center;
}
QPushButton:hover { background-color: #262626; border-style: solid; border-radius: 5px; }
QPushButton:pressed { background-color: #1a1a1a; border-style: solid; border-radius: 5px; }


QMenuBar{
    color: #ffffff;
}

QStatusBar{
    color: #ffffff;
}
/* QListView -----------------------------------------*/
QListView {
    color: #c8c8c8;
    font: 11pt &quot;Consolas&quot;;
	/* 背景色 */
	background-color:#2b2b2b;

	/* 行与行之间交替颜色不同 */
	/* setAlternatingRowColors(true); */
	alternate-background-color:#404040;

	/* 设置是否选中时,整行都高亮显示 */
	show-decoration-selected:1;
}

/* 行与行之间交替颜色不同 */
/* setAlternatingRowColors(true); */
QListView::item:alternate {
	background:#EEEEEE;
}

/* 当条目被选中时 */
QListView::item:selected {
	border:2px solid #6a6ea9;
}

/* 当条目被选中,并且不是有效的 */
QListView::item::selected:!active {
	background:qlineargradient(x1:0, y1:0,x2:1,y2:0, stop:0 #6a6ea9, stop:1 #888dd9);
}

/* 鼠标县浮在条目上 */
QListView::item::hover {
	background:qlineargradient(x1:0, y1:0,x2:1,y2:0, stop:0 #000000, stop:1 #FFFFFF);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,50">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_apply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="Btn_Projects">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Projects</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="Lv_Projects"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="Btn_deleteProject">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_addProject">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>PerforceServer</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="Line_Server"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>ProjectRoot</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="Line_ProjRoot"/>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="Btn_BrowserRoot">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>WatchedPaths</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="Lv_LibPath"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1,1">
             <item>
              <widget class="QPushButton" name="Btn_Search">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Auto Search</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_deleteWatchPath">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_addWatchPath">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QTreeView" name="treeView"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
